CRM/Contribute/BAO/Contribution.php

```diff
@@ -2666,7 +2668,7 @@ public function _gatherMessageValues($input, &$values, $ids = array()) {
             }
             // This is actually used in conjunction with is_quick_config in the template & we should deprecate it.
             // However, that does create upgrade pain so would be better to be phased in.
-            $values['useForMember'] = !$priceSet['is_quick_config'];
+            $values['useForMember'] = !(isset($priceSet) && $priceSet['is_quick_config']);
```

Comment:
Can we initialize `$priceSet = array()` and check for `empty($priceSet['is_quick_config']))` for better readability?